source/kubernetes/appendix.rst

```diff
@@ -266,3 +266,108 @@ Some labels can have multiple values set for them.
         }
 
       .. _`openstack_containerinfra_cluster_v1`: https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs/resources/containerinfra_cluster_v1
+
+.. _k8s-kubelet-reserved:
+
+*******************
+Reserved Resources
+*******************
+
+For Kubernetes to function, the system daemons consumes some vCPU time and
```

Comment:
If daemons is plural then I think it should be "consume".

Reply:
thanks, resolved